src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java

```diff
@@ -70,16 +70,26 @@ public void setFileName(String fileName) {
     fileNameInput.sendKeys(fileName);
   }
 
-  public void clickOnMoreOrFewerOptions() {
+  public void clickOnMoreOptions() {
     moreOrFewerOptions.click();
     waitForValueToBePresentInElementsCssByCss("div.options", "display", "block");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "display" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)